client/src/cmdlfem4x70.c

```diff
@@ -86,12 +86,12 @@ int CmdEM4x70Info(const char *Cmd) {
                   "  ID48 does not use command parity (default).\n"
                   "  V4070 and EM4170 do require parity bit.",
                   "lf em 4x70 info\n"
-                  "lf em 4x70 info -p -> adds parity bit to command\n"
+                  "lf em 4x70 info -x -> adds parity bit to command\n"
                 );
 
     void *argtable[] = {
         arg_param_begin,
-        arg_lit0("p", "parity", "Add parity bit when sending commands"),
+        arg_lit0("x", "parity", "Add parity bit when sending commands"),
```

Comment:
use NULL instead of "x"  and I suggest shorten "parity" to "par"  
 x is used as  "eXecute attack" in other parts of the client.

Reply:
Great suggestion, thanks!